src/lib/Behat/PageObject/ContentStructurePage.php

```diff
@@ -26,22 +26,17 @@ public function __construct(UtilityContext $context)
         $this->rightMenu = new RightMenu($this->context);
     }
 
-    public function createLandingPage(string $name, string $description): void
-    {
-        $this->startCreatingContent('Landing page');
-        $this->context->getSession()->getPage()->findField('Title')->setValue($name);
-        $this->context->getSession()->getPage()->findField('Description')->setValue($description);
-    }
-
     /**
      * Clicks "Create" and selects Content Type in displayed search.
      *
-     * @param $contentType
+     * @param string $contentType
      */
-    public function startCreatingContent(string $contentType): void
+    public function startCreatingContent(string $contentType): UpdateItemPage
     {
         $this->rightMenu->clickButton('Create');
         $this->context->getElementByText($contentType, '.form-check-label')->click();
+
+        return PageObjectFactory::createPage($this->context, UpdateItemPage::PAGE_NAME);
     }
```

Comment:
I was thinking about whether we should write functions in Pages, or just treat it like set of elements and write functions only in Contexts (by calling elements through pages).

Reply:
For me Pages should definitely contain functions: for example editing content (which requires RightMenu and LanguagePicker) should be a function `editContent($language)` which will perform required actions.